src/components/search_bar.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
(Can be over-engineering but) shall we consider adding more properties to the search bar, for example, text placeholder, which is very common in a search bar?

Reply:
Placeholder makes sense, I'll add that.